docs/docs/platform/quickstarts/react.mdx

```diff
@@ -0,0 +1,602 @@
+---
+title: 'Quickstart: React'
```

Comment:
This is not a generic React tutorial but a CRA one, may be good to mention it in the title

Reply:
I think it wouldn't confuse anyone considering that CRA is the basic starting point for React development.

Reply:
Yea if it gets confusing for people in the future we can come back to this.